tests/FINDOLOGIC/Tests/PropertyTest.php

```diff
@@ -27,4 +27,28 @@ public function testAddingMultipleValuesWithoutUsergroupCausesException()
         $property->addValue('foobar');
         $property->addValue('foobar');
     }
+
+    /**
+     * @expectedException \FINDOLOGIC\Export\Data\PropertyKeyNotAllowedException
+     */
+    public function testReservedPropertyKeyOrdernumberCausesException()
+    {
+        $property = new Property('ordernumber');
+    }
+
+    /**
+     * @expectedException \FINDOLOGIC\Export\Data\PropertyKeyNotAllowedException
```

Comment:
It would be nice if a test explicitly checked if the property key shows up in the exception message.

Reply:
Done, used much of your code from above